cli/tests/pcluster/config/test_param_types.py

```diff
@@ -8,12 +8,21 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
+import configparser
 import pytest
 from assertpy import assert_that
 
+import tests.pcluster.config.utils as utils
+from pcluster.config.cfn_param_types import CfnParam, CfnSection, VolumeSizeParam
+from pcluster.config.mappings import EBS
 from pcluster.config.param_types import Param
 
 
+@pytest.fixture()
+def boto3_stubber_path():
+    return "pcluster.utils.boto3"
+
+
```

Comment:
You can get rid of this now that you're calling the function from utils instead of a boto3 client.

Reply:
Right! Done